src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -301,11 +301,11 @@ void EnBom_Move(EnBom* this, PlayState* play) {
 }
 
 void EnBom_WaitForRelease(EnBom* this, PlayState* play) {
-    // if parent is NULL bomb has been released
+    // if parent is NULL if bomb has been released
```

Comment:
Having a hard time parsing this. Should the first if be removed so it reads
"parent is NULL if bomb has been released"?

Reply:
its "if parent pointer is null, the bomb has been released"
I wrote this comment years ago at this point when decompiling the bomb actor lol, so my bad

Reply:
what you said works too if you remove the leading "if"